pkg/reconcilers/deploymentconfig.go

```diff
@@ -140,3 +108,30 @@ func DeploymentConfigContainerResourcesReconciler(desired, existing *appsv1.Depl
 
 	return update
 }
+
+func DeploymentConfigEnvVarMergeMutator(desired, existing *appsv1.DeploymentConfig) bool {
```

Comment:
Not sure about this having the https://github.com/3scale/3scale-operator/pull/548 where it only reconciles the log envvar (https://github.com/3scale/3scale-operator/pull/557).
If we are not sure probably would be a good idea to be safe and just reconcile the worker for now. We can always change to reconcile everything later

Reply:
removing this method for another one focused in a single env var.

Reply:
:+1: